test/fixtures/helper/iam.tf

```diff
@@ -14,13 +14,22 @@
  * limitations under the License.
  */
 
+# Project general bindings
 module "iam_binding_project" {
   source   = "../../../modules/projects_iam"
   mode     = var.mode
   projects = module.base.projects
   bindings = local.project_bindings
 }
 
+# Project conditional bindings
+module "iam_conditional_binding_project" {
```

Comment:
So I added this test and it is failing coz `bindings` are required as per the current implementation. I see two options here:

1.  Add `default = {}` to `bindings` var to cover this test case where user may just start with a conditional binding no general bindings at all. Although it's super rare but may still happen.

2. Remove this test and add `conditional_bindings` to the existing test. We can write a caveat in docs that atleast one general binding is required.

@bharathkkb @morgante  Thoughts?